content/docs/quickstart/hasura.md

```diff
@@ -2,23 +2,43 @@
 title: Run a Hasura App
 ---
 
-Hasura is a cloud-based GraphQL provider for existing databases. This guide will cover zero-coding integration between Neon cloud service and Hasura Cloud. By the end of this guide, we will have a working HTTP API endpoint that uses GraphQL to query our serverless database and responds with a set of rows.
+### Introduction
 
-First, let us set up a Neon project:
+Hasura is a cloud-based GraphQL provider for existing databases. This guide will cover zero-coding integration between Neon cloud service and Hasura Cloud. By the end of this guide, you will have a working HTTP API endpoint that uses GraphQL to query Neon's serverless database and responds with a set of rows.
 
-1. Go to the Neon console <https://console.neon.tech/app/projects>.
-2. Create a new Project or use any existing one.
-3. Open the Project page and click the 'Generate token' button.
-4. Copy DATABASE_URL. We will need it later.
+## Step 1 — Set up a Neon Project
 
-Second, add this Project as a data source in the Hasura Cloud project:
+In this step, you will create or select an existing Neon Project to be used as a data source for Hasura.
 
-1. Go to the Hasura Cloud projects list <https://cloud.hasura.io/projects>.
-2. Create a new project or launch a console for the existing one.
-3. Go to the DATA section, and in the 'Connect Existing Database' tab, paste DATABASE_URL into the corresponding form field. Give it some name and click connect.
-4. That is mostly it! Hasura Cloud will connect and automatically discover the public schema. While Neon will spin up a new compute node for your database when any new connection arrives and suspend it when it is idle.
+First, go to the [Neon console](https://console.neon.tech/app/projects).
 
-Now we can create the first table using the Hasura Console web interface. Let it be table 't' with a single column 'text' of a type 'Text'. Once created, you can put some rows into it and finally navigate to the API section for endpoint creation. In the GraphQL tab, we can query our table with GraphQL, for example:
+Next, create a new Project or use any existing one.
+
+Then, open the Project page, and click the **Generate token** button.
```

Comment:
Outdated. There is no generate token anymore.

It should be something like: i) copy connection (DATABASE_URL) string with button if you created a new project; or ii) reset password / create new role and copy DATABASE_URL if you picked the old one

Reply:
i) copy connection (DATABASE_URL) string with button if you created a new project
or
ii) reset password / create new role and copy DATABASE_URL if you picked the old one

@LizardWizzard @ololobus 

Correct me if I am wrong, but I will need to discuss this in our next meeting to see which directions you would like to replace "Generate token". If you would like to go with the first or second, I am happy to update that in documentation across. Please let me know, thanks!